ZupIT/ritchie-cli
# Added tag update information to repository names in the rit helper

**- What I did**
Added new release notice for added repositories.

> ![image](https://user-images.githubusercontent.com/67295691/94145942-a440cf80-fe49-11ea-9ffd-2e2f55c3d6a6.png)

**- How to verify it**
> Clone this pr with the command
```bash
git fetch upstream pull/541/head:improvement/added-new-version-notice

```
> And add repositories in old versions and use `rit`
```bash
git checkout improvement/added-new-version-notice
make build && cp dist/linux/rit /usr/local/bin/

rit add repo #  Execute with repo https://github.com/brunats/brunats-formulas and version v1.0.3

rit #  Execute and see the notices
```

**- Description for the changelog**

Added tag update information to repository names in the rit helper
